ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -0,0 +1,286 @@
+        }
+    }
+
+    /// Tab bar iems to display in the top section of the side tab bar.
+    /// These TabBarItems don't need landscape images.
+    @objc open var topItems: [TabBarItem] = [] {
+        willSet {
+            willSetItems(in: .top)
+        }
+        didSet {
+            didSetItems(in: .top)
+        }
+    }
+
+    /// Tab bar iems to display in the bottom section of the side tab bar.
+    /// These items do not have a selected state.
+    /// These TabBarItems don't need landscape images.
+    @objc open var bottomItems: [TabBarItem] = [] {
+        willSet {
+            willSetItems(in: .bottom)
+        }
+        didSet {
+            didSetItems(in: .bottom)
+        }
+    }
+
+    /// Selected tab bar item in the top section of the side tab bar.
+    @objc open var selectedTopItem: TabBarItem? {
+        willSet {
+            if let item = selectedTopItem {
+                itemView(with: item, in: .top)?.isSelected = false
+            }
+        }
+        didSet {
+            if let item = selectedTopItem {
+                itemView(with: item, in: .top)?.isSelected = true
+            }
+        }
+    }
+
+    @objc public override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        translatesAutoresizingMaskIntoConstraints = false
+
+        backgroundView.translatesAutoresizingMaskIntoConstraints = false
+        contain(view: backgroundView)
+
+        borderLine.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(borderLine)
+
+        addSubview(topStackView)
+        addSubview(bottomStackView)
+
+        if #available(iOS 13, *) {
+            addInteraction(UILargeContentViewerInteraction())
+        }
+
+        accessibilityTraits = .tabBar
+
+        NSLayoutConstraint.activate([widthAnchor.constraint(equalToConstant: Constants.viewWidth),
+                                     borderLine.leadingAnchor.constraint(equalTo: trailingAnchor),
+                                     borderLine.bottomAnchor.constraint(equalTo: bottomAnchor),
+                                     borderLine.topAnchor.constraint(equalTo: topAnchor)])
+    }
+
+    @available(*, unavailable)
+    @objc public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private enum Section: Int {
+        case top
+        case bottom
+    }
+
+    private struct Constants {
+        static let maxTabCount: Int = 5
+        static let viewWidth: CGFloat = 62.0
+        static let avatarViewSize: CGFloat = 30.0
+        static let avatarViewTopPadding: CGFloat = 18.0
+        static let topStackViewTopPadding: CGFloat = 30.0
+        static let avatarViewTopStackViewPadding: CGFloat = 34.0
+        static let bottomStackViewBottomPadding: CGFloat = 10.0
+        static let topItemSpacing: CGFloat = 38.0
+        static let bottomItemSpacing: CGFloat = 28.0
+        static let itemHeight: CGFloat = 24.0
```

Comment:
curious that you are passing in 28x28 images pdf but the item height is 24?

Reply:
This should be 28, my bad

Reply:
itemHeight will go away once issue#110 gets resolved. Currently, the TabBarItemViews don't have an intrinsic height based on their icon + label so we have to make this assumption about the internal size of the items.